mdata/reorder_buffer_test.go

```diff
@@ -291,21 +291,21 @@ func TestReorderBufferFlushUnsortedData2(t *testing.T) {
 	}
 }
 
-func TestReorderBufferFlushAndHasData(t *testing.T) {
+func TestReorderBufferFlushAndIsEmpty(t *testing.T) {
 	buf := NewReorderBuffer(10, 1)
 
-	if buf.HasData() != false {
-		t.Fatalf("Expected HasData() to be false")
+	if buf.IsEmpty() != true {
```

Comment:
just evaluate buf.IsEmpty directly in all these cases instead of doing comparisons to booleans

Reply:
k, updated again